source/features/conversation-activity-filter.tsx

```diff
@@ -159,6 +159,22 @@ const minorFixesIssuePages = new Set([
 	getRghIssueUrl(4008),
 ]);
 
+function runShortcuts(event: KeyboardEvent): void {
+	if (isEditable(event.target)) {
+		return;
+	}
+
+	if (event.key === 'h' && currentSetting !== 'hideEventsAndCollapsedComments') {
+		currentSetting = 'hideEventsAndCollapsedComments';
+	} else if (event.key === 'f' && currentSetting !== 'hideEvents') {
+		currentSetting = 'hideEvents';
+	} else {
+		currentSetting = 'default';
+	}
```

Comment:
The current logic will disable the filter when any key other than `f` or `h` is pressed.

Instead it should check the key first (maybe with a `switch`) and _then_ toggle the setting (e.g. with a ternary).

Reply:
Ouuu, thx for the catch. Missed it!